data.yml

```diff
@@ -132,6 +132,18 @@ categories:
               summary_integration: "Netbox"
               summary_use_case: "discovery and inventory with a focus on ease of use and accuracy. No coding required."
               summary_business_use_case: "[10 free licence shenanigans] Provides a network inventory and discovery solution that is easy to use and accurate."
+          
+          - name: Netdisco
+            project: full-open-source
+            description: Netdisco is a web-based network management tool suitable for small to very large networks. IP and MAC address data is collected into a PostgreSQL database using SNMP, CLI, or device APIs.
+            homepage_url: http://netdisco.org
+            logo: netbox.jpg
```

Comment:
this should be netdisco.jpg

Reply:
Good spot, pull request #14  put in to fix this.

Reply:
Do I look like I know how to read?